llms/hf_llm/convert.py

```diff
@@ -33,6 +33,22 @@ def fetch_from_hub(hf_path: str):
     )
     return weights, config.to_dict(), tokenizer
 
+def fetch_from_local(local_path):
+    weight_files = glob.glob(f"{local_path}/*.safetensors") + glob.glob(f"{local_path}/*.bin")
+    if len(weight_files) == 0:
+        raise FileNotFoundError("No safetensors or bin files found in {}".format(local_path))
+
+    weights = {}
+    for wf in weight_files:
+        if wf.endswith(".safetensors"):
+            weights.update(mx.load(wf).items())
+        else:
+            weights[Path(wf).name] = mx.ndarray.load(wf)
```

Comment:
I don't think this will run.. `mx.ndarray`?

Reply:
I removed this since only safetensors is supported.